src/pal/inc/pal.h

```diff
@@ -5240,6 +5286,7 @@ PALIMPORT void * __cdecl memcpy(void *, const void *, size_t);
 #endif //defined(_DEBUG)
 PALIMPORT int    __cdecl memcmp(const void *, const void *, size_t);
 PALIMPORT void * __cdecl memset(void *, int, size_t);
+PALIMPORT wchar_t * __cdecl wmemset(wchar_t *, wchar_t, size_t);
```

Comment:
Yep. I gave up the `wmemset` thing. Trying to find another way to improve the perf of converting `0.0`. I found I'm using almost the same code as old implementation except the old one assign '0' to the LPSTR* array, whereas I'm assigning L'0' to wchar_t* array.
https://github.com/dotnet/coreclr/blob/master/src/pal/src/cruntime/misctls.cpp#L187-L190